helpers/index.js

```diff
@@ -11,6 +12,10 @@
    * var app = express();
    * app.use(helpers.errorHandler);
    * */
+
+  var gauge = new client.Gauge('nodejs_errors_total', 'This is a counter of all the exceptions handled.');
```

Comment:
Not sure about the name too. Should probably just be `errors_total{service="front-end,...}`.